helpers/apps

```diff
@@ -111,3 +111,68 @@ ynh_remove_apps() {
 		done
 	fi
 }
+
+# Load an app environment in the current Bash shell
+#
+# usage: ynh_install_apps --app="app"
+#     | arg: -a, --app=     - the app ID
+#
+# Requires YunoHost version 11.0.* or higher.
+ynh_load_app_environment() {
+        # Declare an array to define the options of this helper.
+        local legacy_args=a
+        local -A args_array=([a]=app=)
+        local app
+        # Manage arguments with getopts
+        ynh_handle_getopts_args "$@"
+
+        # Force Bash to be used to run this helper
+        if [[ ! $0 =~ \/?bash$ ]]
+        then
+            ynh_print_err --message="Please use Bash as shell"
+            exit 1
+        fi
+
+        # Make sure the app is installed
+        local installed_apps_list=($(yunohost app list --output-as json --quiet | jq -r .apps[].id))
+        if [[ " ${installed_apps_list[*]} " != *" ${app} "* ]]
+        then
+            ynh_print_err --message="$app is not in the apps list"
+            exit 1
+        fi
+
+        # Make sure the app is installed
+        if ! id -u "$app" &>/dev/null; then
+            ynh_print_err --message="There is no \"$app\" system user"
+            exit 1
+        fi
+
+        # Make sure the app has an install_dir setting
+        local install_dir="$(yunohost app setting $app install_dir)"
+        if [ -z "$install_dir" ]
+        then
+            ynh_print_err --message="$app has no install_dir setting (does it use packaging format >=2?)"
+            exit 1
+        fi
+
+        # Load the Environment variables from the app's service
+        local env_var=`systemctl show $app.service -p "Environment" --value`
+        [ -n "$env_var" ] && export $env_var;
+        export HOME=$install_dir;
+
+        # Source the EnvironmentFiles from the app's service
+        local env_files=(`systemctl show $app.service -p "EnvironmentFiles" --value`)
+        if [ ${#env_files[*]} -gt 0 ]
+        then
+            # set -/+a enables and disables new variables being automatically exported. Needed when using `source`.
+            set -a
+            for file in ${env_files[*]}
+            do
+                [[ $file = /* ]] && source $file
+            done
+            set +a
+        fi
+
+        # Open the app shell
+        su -s /bin/bash $app
```

Comment:
```suggestion
        su -s /bin/bash $app && cd $(systemctl show $app.service -p "WorkingDirectory" --value)
```

Reply:
Good idea :) Why not defaulting to `$install_dir` if there is no service too, like for `mywebapp_ynh`?

Reply:
Good idea!
```suggestion
        local env_dir = $(systemctl show $app.service -p "WorkingDirectory" --value)
        if [[ $env_dir = "" ]];
        then
            env_dir = $install_dir
        fi

        cd $env_dir
        su -s /bin/bash $app
```